packages/bus-query-operation/lib/ActorQueryOperation.ts

```diff
@@ -11,7 +12,9 @@ import type {
   Bindings,
   PatternBindings,
 } from '@comunica/types';
+import type * as RDF from 'rdf-js';
 import type { Algebra } from 'sparqlalgebrajs';
+import * as uuid from 'uuid';
```

Comment:
Can we use a simple counter to keep things more lightweight?

Reply:
`uuid` is already a dependency of `sparqlee` so we're not introducing additional dependencies here. That said, this might change one day so +1 on this, will do.

Reply:
Done!